mautic/mautic
# M3: Mailer Transports - SMTP Transports

**Please be sure you are submitting this against the _staging_ branch.**

[//]: # ( Please answer the following questions: )

| Q  | A
| --- | ---
| Bug fix? | 
| New feature? | 
| Automated tests included? |
| Related user documentation PR URL | 
| Related developer documentation PR URL | 
| Issues addressed (#s or URLs) | https://github.com/mautic/mautic/issues/8018
| BC breaks? | Queued emails from Swiftmailer 5.x are not compatible with Swiftmailer 6.x. Make sure you send all of your queued emails before migrating to M3.
| Deprecations? | "PHP Mailer" removed from the list of supported mailers. It's not supported by the swiftmailer/swiftmailer package anymore.

[//]: # ( Note that all new features should have a related user and/or developer documentation PR in their respective repositories. )

[//]: # ( Required: )
#### Description:
This PR updates swiftmailer/swifmailer package to 6.x.
It also removes `PHP Mailer` from the list of supported mailers, because it is not supported by the swiftmailer/swiftmailer package.

#### Steps to test this PR:
1. Load up [this PR](https://mautibox.com)
2. Go to Email settings.
3. Change your email service provider to be Sendgrid (or some other you like). 
4. Try to send test email.
5. You should see 'Success'
